widget/tabcontainer.go

```diff
@@ -128,6 +128,9 @@ func (c *TabContainer) RemoveIndex(index int) {
 // SetItems sets the container’s items and refreshes.
 func (c *TabContainer) SetItems(items []*TabItem) {
 	c.Items = items
+	if c.current >= len(c.Items) {
+		c.current = -1
+	}
```

Comment:
Perhaps we should select the last item, so we only fallback to -1 if there are no items.
```suggestion
	if l := len(c.Items); c.current >= l {
		c.current = l - 1
	}
```